plugins/spi/SPIOutput.h

```diff
@@ -39,7 +39,9 @@ namespace spi {
 class SPIOutput: public ola::rdm::DiscoverableRDMControllerInterface {
  public:
   // definitions for all SPI Personalities
-  // this hardcoded mapping is needed for backwards compatibility
+  // keep this order - it is important for backwards compatibility!
+  // (the integer representation is used to store the configuration in files.)
```

Comment:
Perhaps also add a note that it's friendly for users if the RDM personality IDs don't change too (well aside from adding more).